packages/compass-e2e-tests/helpers/retry-with-backoff.js

```diff
@@ -14,6 +14,7 @@ async function retryWithBackoff(
       err = e;
       attempt++;
       if (attempt < retries) {
+        console.warn(err);
```

Comment:
Not critical at all, but maybe also use debug here for consistency, what do you think?